sw-precache-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/**.html',
+    'out/static/**.*',
+    'out/**/*.*'
+  ],
+  stripPrefix: 'out/',
+  verbose: true,
+  runtimeCaching: [{
```

Comment:
For URLs in `/2.0.0/_next/`, we can use the `fastest` handler, so that the same build will load faster in subsequent visits.